commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/extension/PageTabFileExt.kt

```diff
@@ -0,0 +1,14 @@
+package ac.uk.ebi.biostd.extension
+
+import ac.uk.ebi.biostd.exception.EmptyPageTabFileException
+import ac.uk.ebi.biostd.integration.SubFormat
+import ac.uk.ebi.biostd.integration.SubFormat.TsvFormat.XlsxTsv
+import ebi.ac.uk.io.ext.size
+import ebi.ac.uk.util.file.ExcelReader.readContentAsTsv
+import java.io.File
+
+internal fun File.readAsPageTab(): String {
+    require(size() > 0) { throw EmptyPageTabFileException(name) }
```

Comment:
I am not sure this is the right place to put this validation IMO serializers should fail if empty content. Meaning xml, json and tsv need to fail if you try to get a submission or file list from empty file. I think that is the bug. Here  it does work but our serialization layer still allow this so if someone call it directly (like the clients) it will still serialize.)

Reply:
Serializers and readers both will fail now with empty content, I'm just trying to make a centralized error for when the user sends an empty file.